src/main/play-doc/operation/CLI.md

```diff
@@ -32,7 +32,7 @@ commands:
 ...
 ```
 
-## Packaging configuration
+## Packaging Configuration
```

Comment:
I think that being consistent is more important than being "correct" here given that being "correct" is worth talking about. :-)

Personally I'd prefer that CSS styling be the means of capitalising a heading as, after all, it is a style.